InformationScripting/src/queries/AstQuery.cpp

```diff
@@ -172,15 +171,14 @@ TupleSet AstQuery::toClassNode(QList<TupleSet> input)
 		return false;
 	};
 
-	auto tuples = ts.tuples(canBeInClass);
+	// TODO currently we remove all the converted nodes, this also means we lose all connections:
```

Comment:
Urgh this comment is still from graph times. With tuples the connections will persist